pywbem_mock/_instancewriteprovider.py

```diff
@@ -277,114 +220,117 @@ def __init__(self, cimrepository=None):
 
     ####################################################################
     #
-    #   CreateInstance server request
+    #   CreateInstance responder
```

Comment:
I try to use the phrase server responder

Reply:
DONE - Changed it consistently to "mock WBEM server responder"